frontend/packages/knative-plugin/src/utils/get-knative-icon.ts

```diff
@@ -23,6 +24,7 @@ export const getKnativeEventSourceIcon = (kind: string): string => {
     case referenceForModel(EventSourceContainerModel):
       return containerSourceImg;
     case referenceForModel(EventSourceCronJobModel):
+    case referenceForModel(EventSourcePingModel):
       return cronJobSourceImg;
```

Comment:
same Icon for both of them?

Reply:
`CronJob` is deprecated with `PingSource` so used same as `CronJob` should go away with next release of Eventing Operator so used the same , can even go with default icon.

Reply:
updated with Default icon